keyboards/converter/a1200/mistress1200/config.h

```diff
@@ -49,6 +49,12 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 #define LED_CAPS_LOCK_PIN B0
 #define LED_PIN_ON_STATE 0
+#define TAPPING_TOGGLE 3
+
+#undef LOCKING_SUPPORT_ENABLE
+#undef LOCKING_RESYNC_ENABLE
+#define NO_MUSIC_MODE
```

Comment:
Audionot enabled, so this wont actually do anything?
```suggestion
```

Reply:
> Audionot enabled, so this wont actually do anything?

As outlined here:
https://docs.qmk.fm/#/squeezing_avr

It seems you need both changes in config.h and in rules.mk, isnt it?

Reply:
Yes, you are right. It does not to anything